cmake/configs/version.cmake

```diff
@@ -1,5 +1,5 @@
-# Generated by dist/s_version. Do not modify.
-set(WT_VERSION_MAJOR 11)
-set(WT_VERSION_MINOR 2)
-set(WT_VERSION_PATCH 0)
-set(WT_VERSION_STRING "WiredTiger 11.2.0: (November 10, 2022)")
+    # Generated by dist/s_version. Do not modify.
```

Comment:
I don't think we should have those spaces here. I will create a ticket to remove those, you should regenerate the release after this is done.

Reply:
We should have had tabs before but somehow we don't. I see that WT-10302 changed the tabs to spaces.

Reply:
Good catch. This change was generated by the `s_all -f` command. The tabs->spaces change might have missed this path.